README.md

```diff
@@ -103,6 +103,31 @@ import { EntityOpsLevelMaturityContent } from '@opslevel/backstage-maturity';
     </EntityLayout.Route>
 ```
 
-## Next Steps
+### Next Steps
 
-Visit the Maturity tab in Backstage to get started.
+Visit the Maturity tab in Backstage to get started. Once you have synced data from Backstage to OpsLevel, you will likely want to set up auto-syncing of data using the [backend-plugin](https://github.com/OpsLevel/backstage-plugin-backend).
+
+
+## Troubleshooting
+
+### 403 API Requests
+
+Please validate that you do not have any middleware set up in Backstage that could be removing cookies or headers from the request by the Backstage proxy.
```

Comment:
don't think we're using cookies for this, but doesn't hurt

Reply:
Removed to reduce confusion.